### PR TITLE
Email error bug fixes

### PR DESCRIPTION
**Pull request by knil-maloon**
The check was incorrect and found a typo

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -546,14 +548,13 @@ function getRequestHeader($regions, $hubData, $vote_grid, $option, $type=""){
     }
     $header_region = '';
     $count_regions = 0;
-
     if($option != '2' && $type != 'home'){
         $small_screen_class = 'hidden-sm hidden-xs';
-        if ($vote_grid == '2') {
-            $header_region .= '<th class="request_grid_icon ' . $small_screen_class . '" style="width:150px" data-sortable="false">' . $current_user['region_code'] . '</th>';
+        if ($vote_grid == '2' || ($isAdmin && $vote_grid == "0")) {
+            $header_region .= '<th class="request_grid_icon ' . $small_screen_class . '" style="width:150px" data-sortable="false">' . $person_region['region_code'] . '</th>';
```
Comment (kcmcg) [inline functions.php:554]: Is $person_region an array or just the region value?

File: HarmonistHubExternalModule.php [hunk 2]
```diff
@@ -221,13 +221,13 @@ function cronMethod($cronAttributes){
                 if(!empty($pidsArray) && !empty($settings)) {
                     try {
                         #CRONS
-                        if ($cronAttributes['cron_name'] == 'cron_metrics' && $settings['deactivate_metrics_cron'][1] != "1") {
+                        if ($cronAttributes['cron_name'] == 'cron_metrics' && $settings['deactivate_metrics_cron___1'] != "1") {
                             include("crontasks/cron_metrics.php");
-                        } else if ($cronAttributes['cron_name'] == 'cron_delete' && ($settings['deactivate_datadown'][1] != "1" || $settings['deactivate_datahub'][1] != "1")) {
+                        } else if ($cronAttributes['cron_name'] == 'cron_delete' && ($settings['deactivate_datadown___1'] != "1" || $settings['deactivate_datahub'][1] != "1")) {
```
[the remark this answers is not in the document]
Comment (knil-maloon) [reply HarmonistHubExternalModule.php:226]: Yes, you are correct although I created a new branch to change something else and somehow all changes from this one https://github.com/vanderbilt-redcap/harmonist-hub/pull/35 are here?

[the remark this answers is not in the document]
Comment (knil-maloon) [reply HarmonistHubExternalModule.php:226]: Maybe I should have checked master fisrt....

File: functions.php (hunk 1 above)
Comment (knil-maloon) [reply functions.php:554]: it's an array of the region

File: HarmonistHubExternalModule.php [hunk 3]
```diff
@@ -221,13 +221,13 @@ function cronMethod($cronAttributes){
                 if(!empty($pidsArray) && !empty($settings)) {
                     try {
                         #CRONS
-                        if ($cronAttributes['cron_name'] == 'cron_metrics' && $settings['deactivate_metrics_cron'][1] != "1") {
+                        if ($cronAttributes['cron_name'] == 'cron_metrics' && $settings['deactivate_metrics_cron___1'] != "1") {
                             include("crontasks/cron_metrics.php");
-                        } else if ($cronAttributes['cron_name'] == 'cron_delete' && ($settings['deactivate_datadown'][1] != "1" || $settings['deactivate_datahub'][1] != "1")) {
+                        } else if ($cronAttributes['cron_name'] == 'cron_delete' && ($settings['deactivate_datadown___1'] != "1" || $settings['deactivate_datahub___1'] != "1")) {
```
Comment (kcmcg) [inline HarmonistHubExternalModule.php:226]: Test Message

**Pull request merged**